src/main/java/bio/terra/cli/app/DockerCommandRunner.java

```diff
@@ -80,19 +80,11 @@ protected int runToolCommandImpl(String command, Map<String, String> envVars)
       bindMounts.put(gcloudConfigDirOnContainer, gcloudConfigDir);
     }
 
-    // check if the system property for testing credentials is populated
-    Optional<Path> credentialsFileForTest = getOverrideCredentialsFileForTesting();
-    if (credentialsFileForTest.isPresent()) { // this is a unit test
-      // mount the file to the container
-      Path adcFileOnContainer = getADCFileOnContainer();
-      bindMounts.put(adcFileOnContainer, credentialsFileForTest.get());
-
-      // set the env var and gcloud auth credentials using the pet SA key file
-      envVars.put("GOOGLE_APPLICATION_CREDENTIALS", adcFileOnContainer.toString());
-      command =
-          "echo \"Setting the gcloud credentials to match the application default credentials\"; "
-              + "gcloud auth activate-service-account --key-file=${GOOGLE_APPLICATION_CREDENTIALS}; "
-              + command;
+    // For unit tests, set CLOUDSDK_AUTH_ACCESS_TOKEN. This is how to programmatically authenticate
+    // as test user, without SA key file
```

Comment:
drop the "without SA key file" part?

Reply:
I think it's important. In general, GCP only documents how to programmatically authenticate with SA key. CLOUDSDK_AUTH_ACCESS_TOKEN isn't documented anywhere.